esrally/driver/runner.py

```diff
@@ -614,17 +619,14 @@ class ForceMerge(Runner):
     async def __call__(self, es, params):
         import elasticsearch
         max_num_segments = params.get("max-num-segments")
-        # preliminary support for overriding the global request timeout (see #567). As force-merge falls back to
-        # the raw transport API (where the keyword argument is called `timeout`) in some cases we will always need
-        # a special handling for the force-merge API.
         request_timeout = params.get("request-timeout")
         mode = params.get("mode")
         merge_params = {"request_timeout": request_timeout}
         if max_num_segments:
             merge_params["max_num_segments"] = max_num_segments
         try:
             if mode == "polling":
-                # we ignore the request_timeout if we are in polling mode and deliberately timeout early
+                # no reason to wait as long as a whole {polling-period} (minimum 1 second)
```

Comment:
What was the reasoning for changing the comment here?

Reply:
it took me a while to understand why 1 was acceptable.  it's only obvious to me now in the diff that i replaced quite a bit of context also.  I will change this to include both comments if that is ok